programs/ziti-edge-tunnel/package/CPackGenConfig.cmake

```diff
@@ -16,6 +16,7 @@ if(CPACK_GENERATOR MATCHES "RPM")
     set(CPACK_RPM_PACKAGE_SUMMARY "OpenZiti Edge Tunneling Software Client")
     set(CPACK_RPM_PACKAGE_URL "https://openziti.github.io/")
     
+    set(CPACK_RPM_PRE_INSTALL_SCRIPT_FILE "${CPACK_RPM_PRE_INSTALL}")
```

Comment:
Should a [pre-dependency](https://cmake.org/cmake/help/latest/cpack_gen/rpm.html#variable:CPACK_RPM_PACKAGE_REQUIRES_PRE) on `shadow-utils` be added for rpm-based distros for the `adduser` command?

Reply:
That looks right to me.

Reply:
To remove the dependency, I now use ```systemd-sysusers```, which is part of the ```systemd``` package.